integrations/acquisition/covidcast/test_dim_tables.py

```diff
@@ -71,19 +73,19 @@ def test_src_sig(self):
         CovidcastRow('src', 'sig', 'day', 'county', 20200414, '11111',
         3, 3, 3, nmv, nmv, nmv, 20200414, 0)
         ]
-        self._db.insert_or_update_bulk(rows)
+        self._db.insert_or_update_batch(rows)
         self._db.run_dbjobs()
         self._db._cursor.execute(self.viewGeoDim)
         record = self._db._cursor.fetchall()
-        geoDimRows = len(list(record))
+        self.geoDimRows = len(list(record))
```

Comment:
Does geoDimRows need to be an object member, or can it be a local variable instead? Remember, [test methods probbly shouldnt alter the TestCase object without a good reason](https://github.com/cmu-delphi/delphi-epidata/pull/925/files/18a27582d8056659a7afa251c78e9e5fef6c7d35..d250f980e070679bcd5f2dfaff70fd1336475ad2#r891518731)